packages/pymedphys_deliverydata/src/pymedphys_deliverydata/dicom/conversion.py

```diff
@@ -97,7 +84,7 @@ def dicom_to_delivery_data(dicom_dataset) -> DeliveryData:
     return merge_delivery_data(delivery_data_by_beam_sequence)
 
 
-def merge_delivery_data(separate: List[DeliveryData]) -> DeliveryData:
+def merge_delivery_data(separate: List[_DeliveryDataBase]) -> _DeliveryDataBase:
```

Comment:
Could do with an explanation of this one one day!

Reply:
The type hints?